<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddAnimal</class>
 <widget class="QDialog" name="AddAnimal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>478</width>
    <height>602</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>180</y>
     <width>70</width>
     <height>23</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="Age">
    <item>
     <widget class="QLabel" name="lbAge">
      <property name="text">
       <string>Age:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="sbAge"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>150</y>
     <width>95</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="Size">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Size:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="cbSize">
      <item>
       <property name="text">
        <string>Teacup</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Small</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Medium</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Large</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>110</y>
     <width>129</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="Breed">
    <item>
     <widget class="QLabel" name="lbBreed">
      <property name="text">
       <string>Breed:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="txtSpeciesBox"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>80</y>
     <width>126</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="Species">
    <item>
     <widget class="QLabel" name="lbSpecies">
      <property name="text">
       <string>Species:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="rbDog">
      <property name="text">
       <string>Dog</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">groupBreed</string>
      </attribute>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="rbCat">
      <property name="text">
       <string>Cat</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">groupBreed</string>
      </attribute>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>40</y>
     <width>129</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="Name">
    <item>
     <widget class="QLabel" name="lbName">
      <property name="text">
       <string>Name:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="txtNameBox"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>210</y>
     <width>145</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="Gender">
    <item>
     <widget class="QLabel" name="lbGender">
      <property name="text">
       <string>Gender:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="rbMale">
      <property name="text">
       <string>Male</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">groupGender</string>
      </attribute>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="rbFemale">
      <property name="text">
       <string>Female</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">groupGender</string>
      </attribute>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>260</y>
     <width>95</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="Fur">
    <item>
     <widget class="QLabel" name="lbFur">
      <property name="text">
       <string>Fur: </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="cbFur">
      <item>
       <property name="text">
        <string>Short</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Long</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Hairless</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="groupBreed"/>
  <buttongroup name="groupGender"/>
 </buttongroups>
</ui>
